src/ome.ts

```diff
@@ -133,10 +133,17 @@ export async function loadPlate(config: ImageLayerConfig, grp: ZarrGroup, plateA
   const { datasets } = imgAttrs.multiscales[0];
   const resolution = datasets[datasets.length - 1].path;
 
+  async function getImgPath(wellPath:string) {
+    // This loads .zattrs for each well but also tries to load .zarray (404) and .zgroup
+    const wellAttrs = await grp.getItem(wellPath).then((g) => g.attrs.asObject());
+    return join(wellPath, wellAttrs.well.images[0].path);
+  }
+  const wellImagePaths = await Promise.all(wellPaths.map(getImgPath));
+
```

Comment:
Since we don't need the group node other than for the attrs, I think we could make a util to handle the specific use case:

```typescript
// src/utils.ts
const decoder = new TextDecoder();
export function getAttrsOnly<T = unknown>(grp: ZarrGroup, path: string) {
  return (grp.store as AsyncStore<ArrayBuffer>)
    .getItem(join(grp.path, path))
    .then((b) => decoder.decode(b))
    .then((text) => JSON.parse(text) as T);
}
```

```typescript

  async function getImgPath(wellPath:string) {
    // This loads .zattrs for each well but also tries to load .zarray (404) and .zgroup
    const wellAttrs = await getAttrsOnly<{ well: Ome.Well }>(grp, wellPath);
    return join(wellPath, wellAttrs.well.images[0].path);
  }
  const wellImagePaths = await Promise.all(wellPaths.map(getImgPath));
```

Reply:
@will-moore any thoughts on this? I am happy to push this PR through and open up a follow up PR regarding this performance enhancement. My main concern is that I don't have many HCS datasets to experiment with, and the IDR links have been somewhat unstable, so it's difficult to test locally.

Reply:
I started to look at this. I know IDR links have been unstable but https://hms-dbmi.github.io/vizarr/v0.1?source=https://s3.embassy.ebi.ac.uk/idr/zarr/v0.1/plates/5966.zarr was working just now (Firefox).
For that big plate it's already quite slow (over a minute) so this might be a killer.
Just trying now with this PR built locally, and it's taking a while (home internet isn't the fastest)!
Nearly 3 mins *before* it even starts to load chunks!
OK, so it finally loaded after 7 minutes (4618 requests). v0.1 vizarr it was 3297 requests and less that 2 mins. But YMMV.
In either case, most users would probably give up since there's no sign of progress.
This plate is probably a bit too ambitious and maybe shouldn't be a blocker if this PR is a critical fix for @camFoltz.
I won't have time to dig any deeper before next week.

Would that performance enhancement help even before this PR?

Reply:
https://hms-dbmi.github.io/vizarr/v0.1?source=https://s3.embassy.ebi.ac.uk/idr/zarr/v0.1/plates/422.zarr worked too, on 2nd attempt (maybe caching is helping).

Reply:
Also happy to share / generate these datasets at will for development purposes